### ISSUE TITLE
v2 sha256sum: unrecognized option: check

### ISSUE THREAD

**Issue by jdebbink** [coverallsapp/github-action#197]
Trying to upgrade to v2 from v1 and getting this error:

```
sha256sum: unrecognized option: check

Usage: sha256sum [-c[sw]] [FILE]...

Print or check SHA256 checksums

	-c	Check sums against list in FILEs
	-s	Don't output anything, status code shows success
	-w	Warn about improperly formatted checksum lines
```

Using the `docker` image.

**Comment by jdebbink** [comment]
Opened https://github.com/coverallsapp/github-action/pull/198 to address

**Comment by afinetooth** [comment]
Closed by #227 

@jdebbink thank you. Left you a comment on your PR [here](https://github.com/coverallsapp/github-action/pull/198#issuecomment-2406220281).
